qml/EditCodeTemplate.qml

```diff
@@ -8,14 +8,23 @@ Column
 
```

Comment:
Nice changes! :+1:  Please fix the indentation though :)

Reply:
Well, basic aim was to get the app fixed. I posted the new version on openrepos and got some questions. Then I decided I didn't want to continue to work on the app and removed it from openrepos and added the code to the original codebase. And that was the latest activity for the app (and this reaction). So do with the pr what you want & good luck with the app ... best regards, rgr

Reply:
I am sorry you feel like that. My aim was not to discourage you to contribute your changes, but rather to make sure that they end up making the app even better.

With just a few fixes we can have your changes merged, and make the app better for everyone, so please don't get discouraged! :)